app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,15 @@
+class OrdersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @orders = current_user.orders.all
```

Comment:
FYI, you don't need to use `#all` after you call `current_user.orders`. This is because `current_user.orders` already returns you all the associated records :)